modules/accounts/accounts.js

```diff
@@ -111,16 +120,30 @@ class Accounts{
         }
 
         const keys = this.getKeys(account);
-        if(!keys) {
-            Logger.error(`Account with this label doesn't exists.`);
+        const contractObj = Helper.findKeyObj(config.contracts, account);
+
+        if(!keys && !contractObj.label && !contractObj.pkh) {
+            Logger.error(`Account with label '${account}' doesn't exists.`);
             return;
         }
 
         try {
             const balance = await RpcRequest.fetchBalance(tezosNode, pkh);
             Logger.info(Helper.formatTez(balance));  
         } catch(error) {
-            Logger.error(`${error}`);
+            if(error.toString().includes(`connect ECONNREFUSED`)) {
+                Helper.errorLogHandler(`Error occured while fetching balance: ${error}`, `Make sure local nodes are in running state....`);
+            } else if(error.toString().includes(`Unexpected end of JSON input`)) {
+                Helper.errorLogHandler(`Error occured while fetching balance: ${error}`, `Make sure account '${account}' is activated on the current provider....`);
+            } else if(error.toString().includes(`Only absolute URLs are supported`)) {
+                Helper.errorLogHandler(`Error occured while fetching balance: ${error}`, `Current provider URL is not supported by network provider....`);
+            } else if(error.toString().includes(`getaddrinfo ENOTFOUND`)) {
+                Helper.errorLogHandler(`Error occured while fetching balance: ${error}`, `Current provider URL is not supported by network provider....`);
+            } else if(error.toString().includes(`Invalid`)) {
+                Helper.errorLogHandler(`Error occured while fetching balance: ${error}`, `Current provider URL is not supported by network provider....`);
+            } else {
+                Logger.error(`Error occured while fetching balance:\n${error}`);
```

Comment:
yes this error will be logged in file. "Helper.errorLogHandler" is being used to store and actual error in file and formatting the error for console so that user will get understandable format of the error. "Logger.error" will also be redirected to the file.